src/test/java/org/opentripplanner/middleware/persistence/PersistenceTest.java

```diff
@@ -19,8 +20,11 @@
     public void canCreateUser() {
         OtpUser user = createUser(TEST_EMAIL);
         String id = user.id;
+        System.out.println("User id:" + id);
```

Comment:
Let's always use LOG.info instead of System.out.println.

Reply:
Removed System.out.println